docs/antctl.md

```diff
@@ -591,11 +591,16 @@ Note: one of `--source` and `--destination` must be a Pod.
 
 The `--flow` (or `-f`) argument can be used to specify the PacketCapture packet
 headers with the [ovs-ofctl](http://www.openvswitch.org//support/dist-docs/ovs-ofctl.8.txt)
```

Comment:
I guess we are not really matching the ovs-ofctl match syntax after all because OVS uses `ip6`, `tcp6`, ...

But your change seems to be consistent with `antctl traceflow`, which is more important